### PR TITLE
Shubham

### PR DESCRIPTION
**Pull request by Shubhambal**

### CODE REVIEW

**Review by sumukh-gn — APPROVED** [review]
Code is reviewed and checked according to guidlines.

File: src/main/java/com/emart/controller/AdminController.java [hunk 1]
```diff
@@ -18,149 +18,191 @@
...
+ * operations.
+ * 
+ * @author Sourabh
+ * @version 3.9.10
+ * @since 24-05-2023
  */
 @RestController
 @CrossOrigin(origins = "*", allowedHeaders = "*")
 @RequestMapping("/admins")
 public class AdminController {
```
Comment (Shubhambal) [inline src/main/java/com/emart/controller/AdminController.java:36]: before adding loggers

File: src/main/java/com/emart/controller/AdminController.java [hunk 2]
```diff
@@ -18,149 +18,191 @@
...
-    // Check if email exists
-    // Parameters:
-    // - admin: The Admin object containing the email to check.
-    public boolean checkEmail(@RequestBody Admin admin) {
-        String email = admin.getEmailId();
-        if (adminService.getByEmail(email) != null) {
-            return true;
-        }
-        return false;
-    }
```
Comment (saurabh7nt) [inline src/main/java/com/emart/controller/AdminController.java:164]: loggers added

**Review by saurabh7nt — COMMENTED** [review]
check loggers as per guidelines

**Pull request merged**